### PR TITLE
Bug/SK-931 | --preferred-combiner should not be boolean

### PR DESCRIPTION
**Pull request by sowmyasris**
Changed --preferred-combiner flag in run_cmd and client_cmd to --combiner
Type change from boolean to string

### CODE REVIEW

File: fedn/network/clients/connect.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class ConnectorClient:
     :param force_ssl: True if https is used, False if http
     :type force_ssl: bool
     :param verify: True if certificate is verified, False if not
-    :type verify: bool
+    :type verify: str
```
Comment (Wrede) [inline fedn/network/clients/connect.py:41]: why is this changed to "str"?

**Review by Wrede — CHANGES_REQUESTED** [review]
I have changed my mind, let keep the name preferred_combiner otherwise it will have conflicts with new feature --combiner and --proxy-host

File: fedn/network/clients/connect.py (hunk 1 above)
Comment (sowmyasris) [reply fedn/network/clients/connect.py:41]: When I tried to install using pip3 install -e .(editable mode), it automatically switches to str, even after I changed to bool

**Review by Wrede — APPROVED** [review]
